commit 3b9ac98c720c7042739eecbabf07767d9eb31632
Author: Volker Krause <vkrause@kde.org>
Date:   Mon Oct 21 17:06:24 2024 +0200

    Add missing dependency on org.kde.coreaddons
    
    Used in the AboutPage, and having it missing here leads to it not being
    bundled on Android, breaking e.g. Itinerary.

diff --git a/src/formcard/CMakeLists.txt b/src/formcard/CMakeLists.txt
index de359ce..4c4fe74 100644
--- a/src/formcard/CMakeLists.txt
+++ b/src/formcard/CMakeLists.txt
@@ -8,6 +8,7 @@ ecm_add_qml_module(formcardplugin
     VERSION 1.0
     GENERATE_PLUGIN_SOURCE
     DEPENDENCIES
+        org.kde.coreaddons
         org.kde.kirigami
         org.kde.kirigamiaddons.delegates
 )
